scripts/cal_diff.py

```diff
@@ -35,10 +35,11 @@ def cmpData(numa, numb):
     args = parser.parse_args()
 
     woq_configs = {
-        "fp32": {"not_quant":True},
-        # "ggml_int4": {"compute_dtype":"int8", "weight_dtype":"int4", "use_ggml":True},
-        "jblas_int4": {"compute_dtype":"int8", "weight_dtype":"int4"},
-        # "jblas_int8": {"compute_dtype":"bf16", "weight_dtype":"int8"},
+            "fp32": {"use_cache":True, "use_quant":False},
```

Comment:
use_cache is removed

Reply:
use_cache became a default parameter？

Reply:
we removed this parameter, local model will be loaded when it exists

Reply:
> use_cache is removed

can't imagine without you :)